lib/appmap/service/integration_test_path_finder.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'appmap/service/test_framework_detector'
+
+module AppMap
+  module Service
+    class IntegrationTestPathFinder
+      class << self
+        def find
+          @paths ||= begin
+            paths = { rspec: [], minitest: [], cucumber: [] }
+            paths[:rspec] = find_rspec_paths if TestFrameworkDetector.rspec_present?
+            paths[:minitest] = find_minitest_paths if TestFrameworkDetector.minitest_present?
+            paths[:cucumber] = find_cucumber_paths if TestFrameworkDetector.cucumber_present?
+            paths
+          end
+        end
+
+        def count_paths
+          find.flatten(2).length - 3
+        end
+
+        private
+
+        def find_rspec_paths
+          find_non_empty_paths(%w[spec/controllers spec/requests spec/integration spec/api spec/features])
```

Comment:
`spec/system` is another good source of integration tests that we should capture.